tests/unittests/BackendTestUtils.h

```diff
@@ -97,20 +97,29 @@ class BackendTest : public BackendStatelessTest {
   Function *F_;
 };
 
-static const auto all_backends = ::testing::Values(
-#ifdef GLOW_WITH_NNPI
-    "NNPI",
-#endif // GLOW_WITH_NNPI
-#ifdef GLOW_WITH_CPU
-    "CPU",
-#endif // GLOW_WITH_CPU
-#ifdef GLOW_WITH_OPENCL
-    "OpenCL",
-#endif // GLOW_WITH_OPENCL
-#ifdef GLOW_WITH_HABANA
-    "Habana",
-#endif // GLOW_WITH_HABANA
-    "Interpreter");
+
+#ifdef GLOW_TEST_BACKEND
```

Comment:
this was done so that the created per-backend test binary won't have Interpreter tests in it (doubles the number of the tests for each backend)